### DISCUSSION TITLE
How to change the default folder path for storing pretrained models

### DISCUSSION THREAD

**Discussion by ADD-eNavarro** [coqui-ai/TTS#2476] [General Q&A] [answered]
I am using Windows, which is important for this question. Also python 3.10, but this shouldn't be important.
I have successfully installed tts and run it, and found that when using pretrained models it stores them in C:/Users/some.user/AppData/Local/tts, but in my case it would be better to be able to configure TTS to save them elsewhere.
I can't find any documentation on how to do this, could anyone point me to the right direction?

Thanks a lot.

**Comment by sgonzalezsilot** [comment]
Any solutions here?

**Reply by eginhard** [reply, marked as answer]
You can now set the `TTS_HOME` or `XDG_DATA_HOME` environment variables with a value of your choice: https://github.com/coqui-ai/TTS/blob/747f688dc374a571e47f2c40358883c7ea11fa32/TTS/utils/generic_utils.py#L129-L131

**Comment by a7med2323** [comment]
you need to add the TTS_HOME to environment variables 
On Windows
For a Single User:

Open the Start Menu and search for "Environment Variables."
Select "Edit the system environment variables."
In the System Properties window, click the "Environment Variables" button.
Under "User variables" or "System variables" (for system-wide setting), click "New" to add a new variable.
Enter TTS_HOME for the variable name and C:\path\to\tts for the variable value.
Click "OK" to save.

On Linux/macOS
For a Single User:

Open a terminal and edit the shell configuration file. Depending on your shell, this might be .bashrc, .bash_profile, .zshrc, etc. For example, to edit .bashrc:



`nano ~/.bashrc`
Add the following line at the end of the file:


`export TTS_HOME="/path/to/tts"`
Save the file and exit the editor (Ctrl+O, Enter to save in nano, and Ctrl+X to exit).

Apply the changes:


`source ~/.bashrc`
